database/seeds/ProjectCommentsTableSeeder.php

```diff
@@ -17,15 +17,17 @@ public function run()
         factory(ProjectComment::class, 1)->create();
 
         //Creates project comments, users, and projects
-        /*$faker = Faker\Factory::create();
+        /*$dir = "public/images/seeds/";
```

Comment:
This whole commented out section is here in case someone wants to have one comment per question per user. They would then only call this seeder and it would set up all the tables.